install/tests/base_test.py

```diff
@@ -1,80 +1,79 @@
         return branch
 
-    def get_os_type(self):
+    @staticmethod
+    def get_os_type():
         os = platform.system()
         if os == "Darwin":
             return "Mac"
         return os
 
-    def os_command(self, command):
-        process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+    @staticmethod
+    def os_command(command):
+        BaseTest.info("Execute : {} ".format(command))
+        process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
         output, error = process.communicate()
-        process.kill()
         return output, error
 
-    def jumpscale_installtion(self, install_type, options=" "):
-        self.info("copy installtion script to /tmp")
-        command = "cp {}/install/jsx.py  /tmp/jsx".format(self.repo_location)
-        self.os_command(command)
+    def docker_command(self, command):
+        command = "docker exec -i {} /bin/bash -c '{}'".format(self.CONTAINER_NAME, command)
+        return BaseTest.os_command(command)
 
-        self.info("Change installer script [/tmp/jsx]to be executed ")
+    @staticmethod
+    def jumpscale_installation(install_type, options=" "):
+        BaseTest.info("copy installation script to /tmp")
+        command = "cp {}/install/jsx.py  /tmp/jsx".format(BaseTest.REPO_LOCATION)
+        BaseTest.os_command(command)
+
+        BaseTest.info("Change installer script [/tmp/jsx] to be executed ")
         command = "chmod +x /tmp/jsx"
-        self.os_command(command)
-        self.info("Configure the non-interactive option")
-        command = "/tmp/jsx configure --no-interactive -s mysecret;"
-        output, error = self.os_command(command)
+        BaseTest.os_command(command)
+        BaseTest.info("Configure the no-interactive option")
+        command = "/tmp/jsx configure --no-interactive -s mysecret"
+        BaseTest.os_command(command)
 
-        self.info("Run script with {} with branch {}".format(install_type, self.js_branch))
-        command = "/tmp/jsx {}  --no-interactive -b {} {}".format(install_type, self.js_branch, options)
-        output, error = self.os_command(command)
+        BaseTest.info("Run script with {} with branch {}".format(install_type, BaseTest.get_js_branch()))
+        command = "/tmp/jsx {}  --no-interactive -b {} {}".format(install_type, BaseTest.get_js_branch(), options)
+        output, error = BaseTest.os_command(command)
         return output, error
 
-    def check_js_container_installtion(self):
-        self.info(" Check that js container exist ,should succeed")
+    @staticmethod
+    def is_js_container_installed():
+        BaseTest.info(" Check that js container exist ,should succeed")
         command = "/tmp/jsx container-kosmos"
-        output, error = self.os_command(command)
-        if error:
-            output, error = self.jumpscale_installtion("container-install", "-n {}".format(self.js_container))
```

Comment:
need jumpscale installtion if not exist here

Reply:
according to the new setup and teardown methods, there is no test case calling this method anymore.